### DISCUSSION TITLE
Trying to replicate snapshot config in Ubuntu derivatives

### DISCUSSION THREAD

**Discussion by ObsidianRazor** [SpiralLinux/SpiralLinux-project#454] [Q&A]
Hi! Big fan of this project, and I'm trying to get snapshots working in a similar fashion in Ubuntu derivatives like Pop Os and KDE neon.

Is there any documentation on how to set up volumes so grub-btrfs can boot into snapshots?

It's the only piece I'm missing and for the life of me I cannot make it work!

Thank you!

**Comment by marwenius** [comment]
[Maybe one of the links mentioned there will help? (e.g. the last link)](https://github.com/orgs/SpiralLinux/discussions/58#discussioncomment-3891503)

**Reply by ObsidianRazor** [reply]
Going to give them a look, thanks!
